### ISSUE TITLE
SP5 WPF: "Copy to Clipboard" is broken

### ISSUE THREAD

**Issue by MCF** [ScottPlot/ScottPlot#3214]
As per title, copy to clipboard with the WPF default context menu does not copy the image to the clipboard.  I've  tested the WinForms context menu and that one will copy to the clipboard.   The copy to clipboard functions are fairly different between the two.

https://github.com/ScottPlot/ScottPlot/blob/54c172f903553186b4516db07c7c272605ed8f4c/src/ScottPlot5/ScottPlot5%20Controls/ScottPlot.WPF/WpfPlotMenu.cs#L104-L109

https://github.com/ScottPlot/ScottPlot/blob/54c172f903553186b4516db07c7c272605ed8f4c/src/ScottPlot5/ScottPlot5%20Controls/ScottPlot.WinForms/FormsPlotMenu.cs#L42-L47

ScottPlot NuGet version: **5.0.16**

I've tested this  in a few applications and it fails in all of them.  I threw together the basic scatter plot example FWIW and it fails with that one for example:

```xaml
<Window x:Class="SPCopyToClipBoardTest.MainWindow"
        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
        xmlns:ScottPlot="clr-namespace:ScottPlot.WPF;assembly=ScottPlot.WPF"
        xmlns:local="clr-namespace:SPCopyToClipBoardTest"
        mc:Ignorable="d"
        Title="MainWindow" Height="450" Width="800">
    <Grid>
        <ScottPlot:WpfPlot x:Name="MyPlot" />
    </Grid>
</Window>
```

```cs
using ScottPlot;
using System.Windows;

namespace SPCopyToClipBoardTest
{
    /// <summary>
    /// Interaction logic for MainWindow.xaml
    /// </summary>
    public partial class MainWindow : Window
    {
        public MainWindow()
        {
            InitializeComponent();

            Coordinates[] coordinates =
            {
                new(1, 1),
                new(2, 4),
                new(3, 9),
                new(4, 16),
                new(5, 25),
            };

            MyPlot.Plot.Add.Scatter(coordinates);
        }
    }
}
```

**Comment by swharden** [comment]
Thanks for reporting this @MCF! I appreciate the links to the relevant code too. This seems to work! I'll add it to the next release 🚀 

```cs
public static void CopyImageToClipboard(IPlotControl plotControl)
{
    PixelSize lastRenderSize = plotControl.Plot.RenderManager.LastRender.FigureRect.Size;
    Image bmp = plotControl.Plot.GetImage((int)lastRenderSize.Width, (int)lastRenderSize.Height);
    byte[] bmpBytes = bmp.GetImageBytes();

    using MemoryStream ms = new();
    ms.Write(bmpBytes, 0, bmpBytes.Length);
    BitmapImage bmpImage = new();
    bmpImage.BeginInit();
    bmpImage.StreamSource = ms;
    bmpImage.EndInit();
    Clipboard.SetImage(bmpImage);
}
```

**Comment by swharden** [comment]
resolved in 924a041baf130669d66ec2f4343c2f011082b8a6
